src/Tribe/Attendee_Repository.php

```diff
@@ -88,6 +106,21 @@ public function __construct() {
 			'rsvp_status'           => [ $this, 'filter_by_rsvp_status' ],
 		] );
 
+		// Add object default aliases.
+		$this->update_fields_aliases = array_merge( $this->update_fields_aliases, [
```

Comment:
* :no_entry_sign: Opening parenthesis of a multi-line function call must be the last content on the line
* :no_entry_sign: Only one argument is allowed per line in a multi-line function call

<sub>hash: a91ea2ddc83bda8cf18b730288429503</sub>